src/Engine/Manager/RenderManager.cpp

```diff
@@ -89,6 +89,8 @@ void RenderManager::Render(World& world, bool soundSources, bool particleEmitter
             { GPUPROFILE("Render main window", Video::Query::Type::TIME_ELAPSED);
                 const glm::mat4 projectionMatrix = camera->GetComponent<Lens>()->GetProjection(mainWindowRenderSurface->GetSize());
                 const glm::mat4 viewMatrix = glm::inverse(camera->GetModelMatrix());
+                const glm::vec3 position = camera->GetWorldPosition();
+                const glm::vec3 up(viewMatrix[0][1], viewMatrix[1][1], viewMatrix[2][1]);
```

Comment:
Can be changed into up = camera->GetWorldOrientation() * glm::vec3(0, 1, 0); 
Once the quaternion refactoring branch is merged

Reply:
const glm::vec3 up(viewMatrix[0][1], viewMatrix[1][1], viewMatrix[2][1]); is faster